src/content/docs/en/install-and-setup.mdx

```diff
@@ -57,201 +57,105 @@ Astro is built with Vite which targets browsers with modern JavaScript support b
       </Fragment>
     </PackageManagerTabs>
 
-    You can run `create astro` anywhere on your machine, so there's no need to create a new empty directory for your project before you begin. If you don't have an empty directory yet for your new project, the wizard will help create one for you automatically.
+    If all goes well, you will see a success message followed by some recommended next steps.
+    
+2. Now that your project has been created, you can `cd` into your new project directory to begin using Astro.
 
-    If all goes well, you will see a success message followed by some recommended next steps. Now that your project has been created, you can `cd` into your new project directory to begin using Astro.
+3. If you skipped the "Install dependencies?" step during the CLI wizard, then be sure to install your dependencies before continuing.
 
-2. If you skipped the "Install dependencies?" step during the CLI wizard, then be sure to install your dependencies before continuing.
-
-3. You can now [start the Astro dev server](#start-the-astro-dev-server) and see a live preview of your project while you build!
-</Steps>
-
-### Use a theme or starter template
-
-You can also start a new astro project based on an [official example](https://github.com/withastro/astro/tree/main/examples) or the `main` branch of any GitHub repository by passing a `--template` argument to the `create astro` command.
-
-<Steps>
-1. Explore our [themes and starters showcase](https://astro.build/themes/) where you can browse themes for blogs, portfolios, documentation sites, landing pages, and more! Or, [search on GitHub](https://github.com/search?o=desc&q=astro+starter&s=stars&type=Repositories) for even more starter projects.
-
-2. Run the following command in your terminal, substituting the official Astro starter template name, or the GitHub username and repository of the theme you want to use:
-
-    <PackageManagerTabs>
+        <PackageManagerTabs>
       <Fragment slot="npm">
       ```shell
-      # create a new project with an official example
-      npm create astro@latest -- --template <example-name>
-
-      # create a new project based on a GitHub repository’s main branch
-      npm create astro@latest -- --template <github-username>/<github-repo>
+      npm install astro
```

Comment:
When we skip the "Install dependencies" step, we still have a `package.json` file and `astro` is still in the `dependencies`, so I think we just need:
```suggestion
      npm install
```

Reply:
Great call!